modules/ROOT/pages/managing-instances/instance-actions.adoc

```diff
@@ -187,3 +187,16 @@ Type the exact name of the instance (as instructed) to confirm your decision, an
 ====
 There is no way to recover data from a deleted Aura instance.
 ====
+
+== Mark an instance as production
+
+label:AuraDB-Virtual-Dedicated-Cloud[]
+label:AuraDB-Business-Critical[]
+
+Admins can mark a database instance as `production` in AuraDB Virtual Dedicated Cloud and AuraDB Business Critical.
+
+When Neo4j updates Aura Database Versions, instances marked as `production` are last to have the Aura Database Version updated. Because updates are applied to these database instances after other database instances recieve updates, you can monitor any potential impact on less critical databases first.
+
+After marking the instance as `production` the label is applied immediately, and all instance actions (such as pause or clone) will be temporarily unavailable while the instance is set to `production` on the Neo4j side. This should only take a few moments.
```

Comment:
```suggestion
After marking the instance as `production`, the label is applied immediately, and all instance actions (such as pause or clone) are temporarily unavailable while the instance is set to `production` on the Neo4j side. 
```

Reply:
What do you mean with _on the Neo4j side_?

Reply:
Whilst Aura updates the instance to `production` in the Aura backend.